yarn-project/noir-protocol-circuits/src/crates/private-kernel-lib/src/private_kernel_inner.nr

```diff
@@ -458,15 +451,12 @@ mod tests {
     fn input_validation_malformed_arrays_read_requests() {
         let mut builder = PrivateKernelInnerInputsBuilder::new();
 
-        builder.private_call.public_inputs.read_requests.push_array([
-            SideEffect {
-            value: 0,
-            counter: 0
-            },
-            SideEffect {
-                value: 9123,
-                counter: 1,
-                }]);
+        builder.private_call.public_inputs.read_requests.push_array(
```

Comment:
Another case where the formatting looks off. It seems to be when we got arrays inside other structs or as part of the arguments, the ending `]` is located weirdly.